src/cascadia/TerminalSettingsEditor/EditColorScheme.xaml

```diff
@@ -482,7 +489,14 @@
                         </Flyout>
                     </Button.Flyout>
                 </Button>
-            </Border>
+                <TextBlock x:Name="SelectedSchemeDisclaimer"
+                           x:Uid="ColorScheme_DeleteButtonDisclaimerInBox"
+                           Grid.Column="1"
+                           VerticalAlignment="Center"
+                           Style="{StaticResource DisclaimerStyle}"
+                           TextWrapping="Wrap"
+                           Visibility="{x:Bind ViewModel.IsInBoxScheme, Mode=OneTime}" />

```

Comment:
```suggestion
                           Visibility="{x:Bind ViewModel.IsInBoxScheme}" />
```
smallest nit: I think you can remove the mode here because `OneTime` is the default.